docs/changelog.md

```diff
@@ -1,4 +1,19 @@
 # TBD
+### Features
+* Add `SharedDirectory` object which contains two fields `absFilepathOnThisContainer` and `absFilepathOnServiceContainer` that store the values of the service directory absolute path in the testsuite container and the service container
+* Update API container service proto definition adding the `relative_service_dirpath` field in `RegisterServiceResponse` and `GetServiceInfoResponse`
+
+### Breaking Changes
+* Mix `ContainerCreationConfig` and `ContainerRunConfig` in the new `ContainerConfig` which contains all the necessary information used to create and run a service container
+  * Users should replace the creation of `ContainerCreationConfig` and `ContainerRunConfig` objects with an implementation of an anonymous function that dynamically generates the new `ContainerConfig` object
+* Update `NetworkCtx.AddService()` method adding the `kurtosisEnclaveDataVolMountpointOnServiceContainer` param and replacing the `containerCreationConfig` param and the `generateRunConfigFunc` with the `containerConfigSupplier` param
+  * Users should update `NetworkCtx.AddService()` calls passing it now the `kurtosisEnclaveDataVolMountpointOnServiceContainer` which is the root mountpoint name in the service container and the `containerConfigSupplier` which is 
```

Comment:
s/is the root mountpoint name in the service container/is the location where the Kurtosis volume should be mounted on the service container/